### PR TITLE
Feature add advanced search

### PR DESCRIPTION
**Pull request by GuenoleK**
### Description

Preparing the advanced-search components' example for the showcase

### CODE REVIEW

File: src/page/search/advanced-search/example/index.jsx [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+let crit = null;
+const getSearchService = (scoped) => {
+    return (criteria) => {
+        console.log(criteria);
+        crit = criteria
+        return new Promise(success => {
+            setTimeout(() => {
+                const groups = {
+                    France: [
```
Comment (pierr) [inline src/page/search/advanced-search/example/index.jsx]: `['Paris', 'Lyon'].map( (ville, idx) => {id: idx, name: ville})`

File: src/page/search/advanced-search/example/index.jsx [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+    }
+};
+
+const service = {
+    unscoped: getSearchService(false),
+    scoped: getSearchService(true)
+};
+
+const Line = React.createClass({
+    mixins: [FocusComponents.list.selection.line.mixin],
```
Comment (pierr) [inline src/page/search/advanced-search/example/index.jsx]: Outside the React.createClass `const linePreset = FocusComponents.list.selection.line.mixin]`

**Comment by GuenoleK** [comment]
Gotta had a good capture screenshot before the merge

**Comment by Bernardstanislas** [comment]
Waiting for the capture to merge then

**Comment by GuenoleK** [comment]
Job done

**Pull request merged**